### DISCUSSION TITLE
Is it possible to create a custom MQTT cmnd?

### DISCUSSION THREAD

**Discussion by guydvir2** [arendst/Tasmota#11324] [Q&A] [answered]
Is it possible to create your own MQTT commands and create a rule to create a function when such command is sent ?

**Comment by barbudor** [comment, marked as answer]
You should look at EVENT

You can publish a command to `cmnd/topic/event` with payload `myevent somevalue`
And catch in a rule
`on event#myevent do something %value% endon`

Read the Rules section and Event command for more details

**Comment by sfromis** [comment]
While `Event`is indeed the easiest way of creating commands, the compile-time optional [Subscribe](https://tasmota.github.io/docs/Subscribe-&-Unsubscribe/) feature will allow you to choose command names for events, instead of needing to go through the event command.
